resources/build/Makefile

```diff
@@ -11,7 +11,7 @@ default: docker-image
 ##
 
 # This container version mimicks the VFX reference platform versioning.
-CONTAINER_VERSION=2022.1
+CONTAINER_VERSION=2022.2
```

Comment:
[fix] As this has been published, should this be .3? (and do we need a commit somewhere with as it was for the published .2 container)

Reply:
It was published as .1, I bumped to .2.  I overwrote the previous published .2 with the downgraded fmtlib, reasoning that it's unlikely anyone pulled .2 in those couple of hours.

Or do you mean we should bump to the not-yet-released future version inside the Makefile?

Reply:
If you've already overwritten the previous publish, then its moot, (good to avoid that where we can next time).